benchmark/fft/fft_config.cc

```diff
@@ -0,0 +1,46 @@
+#include "benchmark/fft/fft_config.h"
+
+#include <algorithm>
```

Comment:
Please remove it.

Reply:
cpp lint is spitting error message for
```c++
base::ranges::sort(exponents_);
```
this code, so I'll just add `// NOLINT` to the line.